### PR TITLE
add installation guide to readme

### LINKED ISSUE #654: Install guide in readme.md
### OpenTofu Version

```shell
v1.6.0-alpha1 and onwards
```

### Use Cases

# Install
When wanting to install openTofu on your client you can easily have a look at the readme.md to install the binary.

### Attempted Solutions

One solution right now is to install from a git clone with the earlier description where you install Go and it's dependencies. For a newcomer there may me easier to have a command to input in the command-line to install the binary.

### Proposal

# Install guide
Short description on how to install the binary for the different OS's.

### PR DESCRIPTION
**Pull request by kmgrime**
Resolves #654

## Target Release

1.6.0

### NEW FEATURES | UPGRADE NOTES | ENHANCEMENTS | BUG FIXES | EXPERIMENTS

 

-  Updates main readme.md with installation guide
-  Minor syntax cleanup on other bullets existing.

### CODE REVIEW

**Comment by kislerdm** [comment]
@kmgrime Hey! Could you please review the Developer Certificate of Origin (DCO) and accept it by signing your commit? Please find details in the [contributions guidelines](https://github.com/opentofu/opentofu/blob/main/CONTRIBUTING.md#contributing-a-code-change). Thanks!

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # OpenTofu
 
-- Manifesto: https://opentofu.org/manifesto
-- About the OpenTofu fork: https://opentofu.org/fork
+- [Manifesto](https://opentofu.org/manifesto)
+- [About the OpenTofu fork](https://opentofu.org/fork)
+- [Installation](https://opentofu.org/docs/intro/install)
```
Comment (kislerdm) [inline README.md:5]: **(suggestion)**

```suggestion
- [How to install](https://opentofu.org/docs/intro/install)
```

**Review by kmgrime — COMMENTED** [review]
Looks good!

**Comment by kmgrime** [comment]
@kislerdm Do I make another commit with sign, or is it possible to sign already pushed commits?

**Comment by serdardalgic** [comment]
@kmgrime https://stackoverflow.com/a/15667644/566715

**Comment by kmgrime** [comment]
Signed off! Sorry about that - but learned alot :)

**Review by kislerdm — APPROVED** [review]

**Pull request merged**